apps/asap-server/src/controllers/teams.ts

```diff
@@ -237,26 +224,23 @@ export default class Teams implements TeamController {
 
   async fetchById(teamId: string, user: User): Promise<TeamResponse> {
     const query = buildGraphQLQueryFetchTeam();
-    const teamPromise = this.client.request<ResponseFetchTeam, { id: string }>(
-      query,
-      { id: teamId },
-    );
-
-    const usersPromise = fetchUsers(teamId, this.users.client);
-
-    const [teamResponse, users] = await Promise.all([
-      teamPromise,
-      usersPromise,
-    ]);
+    const teamResponse = await this.client.request<
+      ResponseFetchTeam,
+      { id: string }
+    >(query, { id: teamId });
 
     const { findTeamsContent: team } = teamResponse;
+
     if (!team) {
       throw Boom.notFound();
     }
 
     return transformGraphQLTeam(
       team,
-      transformRestTeamMember(users, teamId),
+      transformGraphQLUser(
```

Comment:
same as in my other comment - whats the point of passing users as a separate argument if the `referencingUsersContents` property is accessible from the `team`?